linkerd/app/outbound/src/http/logical.rs

```diff
@@ -99,53 +133,110 @@ impl<N> Outbound<N> {
     }
 }
 
-// === impl ProfileRoute ===
+// === impl Params ===
 
-impl svc::Param<profiles::http::Route> for ProfileRoute {
-    fn param(&self) -> profiles::http::Route {
-        self.route.clone()
+impl From<(Profile, Logical)> for Params {
```

Comment:
nit: _maaaaaybe_ worth a comment saying where this gets impl gets consumed? not a huge deal though.